src/entry-point-client/index.tsx

```diff
@@ -1,20 +1,51 @@
+import '@babel/polyfill';
```

Comment:
can we do 
```suggestion
import "core-js/stable";
import "regenerator-runtime/runtime";
```
instead per [recommendation](https://babeljs.io/docs/en/babel-polyfill) or does that break the Auth0 integration?

Reply:
Turns out we only need the regenerator-runtime import, not the core-js import. Thanks!